schemas/stsci.edu/asdf/transform/box1d-1.0.0.yaml

```diff
@@ -0,0 +1,43 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/box1d-1.0.0"
+tag: "tag:stsci.edu:asdf/transform/box1d-1.0.0"
+title: >
+  One dimensional box model.
+
+description: >
+  One dimensional box.
+
+examples:
+  -
+    - A 1D box of width 4.0, amplitude 10.0, centered at x=1.5. 
+    - |
+      !transform/box1d-1.0.0
+        amplitude: 10.0
+        bounding_box: [-0.5, 3.5]
```

Comment:
should I go ahead make 1.3.0 and then have just the new models use this version? also, why is 1.1.0 the exact same as 1.0.0 (similarly, when does it make sense to make a new version of transform?)

Reply:
@nden what do you think?  I'm inclined to say yes, just so we don't have to release the standard again right away.

I don't know why 1.1.0 was created with no changes -- it could be that at the time we assumed that the schema versions had to stay identical to the standard version.